docs: DOC-1757: Self-Hosted Anonymous SMTP Configuration (#6604)
spectrocloud/librarium: _partials/self-hosted/_smtp.mdx

    | **Field**                    | **Description**                                                                                                                                                                           | **Required** |
    | ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
    | **Insecure Skip TLS Verify** | This option disables Transport Layer Security (TLS) certificate verification. Enable this option if your SMTP server is using a self-signed certificate or has a certificate that is not trusted by the system.  | No |
    | **Outgoing Server**          | The host name of the SMTP server. For example, `smtp.gmail.com`.                                                                                                                           | Yes |
    | **Outgoing Port**            | The port number of the SMTP server.                                                                                                                                                       | Yes |

>     | **From Email**               | The email address from which emails will be sent. Emails sent from {props.version} will be sent from this email address.                                                                      | Yes |

Reviewer: ```suggestion
    | **From Email**               | The email address from which emails will be sent. Emails sent by Palette will use this email address as the sender.                                                                      | Yes |
```

Author: So, I have to keep {props.version}, as this is part of a partial. The markdown file sets props.version to either "Palette" or "VerteX," depending on the page. So, when you view the page, the correct product will be substituted.  

I agree with general rewording, though! I will make that change :)

>     | **Username**                 | The username of the SMTP server. This is required for authenticated SMTP. If you are using anonymous SMTP, do not include a username or password.                                                                                                                                                         | No |

Reviewer: ```suggestion
    | **Username**                 | The username of the SMTP server. If you are using anonymous SMTP, do not include a username or password.                                                                                                                                                         | Yes, for authenticated SMTP. No, anonymous SMTP.|
```
Not sure about this one though.

Author: It is not technically required, so I am going to leave this one as is. We want to reserve required for "if you don't have anything here, you cannot proceed," and that is not the case here.

>     | **Password**                 | The password of the SMTP server. This is required for authenticated SMTP. Certain SMTP servers, such as `smtp.gmail.com`, require app passwords instead of user passwords. If you are using anonymous SMTP configuration, do not include a username or password.                                                                                                                                                         | No |

Reviewer: ```suggestion
    | **Password**                 | The password of the SMTP server. Certain SMTP servers, such as `smtp.gmail.com`, require app passwords instead of user passwords. If you are using anonymous SMTP configuration, do not include a username or password.                                                                                                                                                         | Yes, for authenticated SMTP. No, anonymous SMTP.|
```
Similar to teh previous suggesyions.

Author: See above.

> 5. Select **Validate configuration**. A success message is displayed if the configuration is valid; otherwise, an error message is displayed. Additional error details can be found in the {props.version} `system-<ID>-<ID>` container logs, located in the `hubble-system` namespace.

Reviewer: ```suggestion
5. Select **Validate configuration**. A success message is displayed if the configuration is valid; otherwise, an error message is displayed. For additional error details, refer to the {props.version} `system-<ID>-<ID>` container logs, located in the `hubble-system` namespace.
```
To avoid passive voice

6. **Save** your changes.

    If you are using anonymous SMTP, a security message is displayed. If possible, update your configuration to use authenticated SMTP. The system console reminds you that you are using an anonymous SMTP configuration by displaying a warning icon beside **Connection**.

## Validate

> The SMTP configuration is validated when you select **Validate configuration**. A success message is displayed if the configuration is valid; otherwise, an error message is displayed. Additional error details can be found in the {props.version} `system-<ID>-<ID>` container logs, located in the `hubble-system` namespace.

Reviewer: ```suggestion
The SMTP configuration is validated when you select **Validate configuration**. A success message is displayed if the configuration is valid; otherwise, an error message is displayed. For additional error details, refer to the {props.version} `system-<ID>-<ID>` container logs, located in the `hubble-system` namespace.
```

Author: Fixing this as well :)
